### ISSUE TITLE
📌 Community-crowdsourced Horizon instance URLs

### ISSUE THREAD

**Issue by JFWooten4** [stellar/stellar-docs#1127]
Similar to the list of [RPC providers](https://developers.stellar.org/docs/data/rpc/rpc-providers#publicly-accessible-rpc-urls), might we curate some notable public Horizon servers available for starting developers? Having a place to query live data was quite helpful for me starting out, and this could be something harder to find now that the SDF's deprecating some of these ops while rolling out Hubble. Perhaps we could add them to [this page](https://github.com/stellar/stellar-docs/blob/main/docs/data/horizon/horizon-providers.mdx), so that the default mantra for new users isn't just "spin up your own instance" right off the bat?

[<img width="640" alt="few-server-options" src="https://github.com/user-attachments/assets/3d29c462-2d42-4fc1-812b-e53e444e41d7">](https://publicnode.org/)

**Comment by ElliotFriend** [comment]
@JFWooten4 Since we merged #1145, can this issue be closed? Or, do you still have more work to do towards this end?

**Comment by JFWooten4** [comment]
Yes, my bad. I wish you didn't need write access to associate issues with PRs.
